addons/storyshots/src/test-bodies.js

```diff
@@ -16,7 +16,7 @@ function getSnapshotFileName(context) {
   }
 
   const { dir, name } = path.parse(fileName);
-  return path.format({ dir, name, ext: '.storyshot' });
+  return path.format({ dir: path.join(dir, '__snapshots__'), name, ext: '.storyshot' });
```

Comment:
Hmmm @ndelangen @igor-dv @tmeasday is this actually an improvement? I think I liked it better before...

Reply:
I don't know how to feel about it.. Even though it's a "default" Jest behavior I would like to see `.storyshots` here (I like the webstrom grouping feature 😄): 
![1](https://user-images.githubusercontent.com/7867954/29747528-8a86dad0-8b06-11e7-99f6-bccbb89ca891.png)

Maybe to make it configurable ? 

Nevertheless I want to add a test as part of the Storyshots that will check if every `.storyshot` file has its story file (Like jest does with the `.snap` files)